### PR TITLE
reporting

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

**Review by croqueGrec09 — COMMENTED** [review]
one remark

File: grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -3230,15 +3230,38 @@ reporting.macro.info1 = Erstellen Sie eine Suchanfrage anhand kombinierbarer Fil
...
+reporting.filter.selection = Selection
+reporting.filter.no.matches = No results were found with these filter settings.
+reporting.query.chart = Visualization
+reporting.details.property.value = Property value
+
+reporting.modal.error = Unknown error.
+reporting.modal.nodata = No results found.
+reporting.modal.nodetails = Unfortunately no details are available.
+
+reporting.baseConfig.legalInfo.label = Erstellt bzw. organisiert durch ..
```
[the remark this answers is not in the document]
Comment (klodav) [reply grails-app/i18n/messages.properties:3243]: yes

[the remark this answers is not in the document]
Comment (croqueGrec09) [reply grails-app/i18n/messages.properties:3243]: Alright ...

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**